### PR TITLE
build: check if mirinae package is npm linked in vite serve process & update mirinae usages

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [x] Refactor
- [x] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description

- Add optimizeDeps for npm linked case
  - Not always use this property. It is inserted only when the mirinae is npm linked and when is serve mode.
- Update mirinae version and some updated usages

### CODE REVIEW

File: vite.config.js [hunk 1]
```diff
@@ -1,24 +1,61 @@
...
+import fs from 'fs';
 
 import vuePlugin from '@vitejs/plugin-vue2';
 import { defineConfig, loadEnv } from 'vite';
 import StylelintPlugin from 'vite-plugin-stylelint';
-import VueTypeImports from 'vite-plugin-vue-type-imports'
+import VueTypeImports from 'vite-plugin-vue-type-imports';
 
-export default defineConfig(({ command, mode }) => {
+const isPackageLinked = (packageName) => new Promise((resolve, reject) => {
```
Comment (yuda110) [inline vite.config.js:11]: praise: 👍

**Review by yuda110 — APPROVED** [review]
wow...LGTM!!

**Review by minseolee — APPROVED** [review]
LGTM!

**Review by piggggggggy — APPROVED** [review]
LGTMmmm!!

**Pull request merged**